assets/queries/openAPI/2.0/basepath_with_wrong_format/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	not regex.match("^/", doc.basePath)
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("basePath={{%s}}", [doc.basePath]),
```

Comment:
Since `basePath` is just a string, not an array, I think search key can be just `basePath`, as following:
```suggestion
		"searchKey": "basePath",
```

Reply:
This is to avoid cases like comments or names that contain basePath where the search key will point to the wrong lines, for example:
```
1: {
2:  "swagger": "2.0",
3:  "info": {
4:    "title": "basePath API Overview",
5:    "version": "1.0.0"
6:  },
7: "basePath": "/api"
8:}
```
The search key, in this case, will point to line 4 which is not correct.
This is more to avoid further problems